examples/App/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": ["babel-preset-expo"],
+  "env": {
+    "development": {
+      "plugins": ["transform-react-jsx-source",
+        ["module-resolver", {
+          "alias": {
+            "react-router-native-stack": "../.."
```

Comment:
I followed steps [here](https://blog.callstack.io/adding-an-example-app-to-your-react-native-library-d23b9741a19c) to resolve "multiple require"-style errors and make this example `App` use imports the same way a "real" app using `react-router-native-stack` would do

Reply:
Nice!